### ISSUE TITLE
About WGAN-div loss function

### ISSUE THREAD

**Issue by Lmy0217** [bojone/gan#4]
In [wgan_div_celeba.py#L129](https://github.com/bojone/gan/blob/master/keras/wgan_div_celeba.py#L129), there is `d_train_model.add_loss(d_loss + grad_loss)` and `d_loss` is defined by `K.mean(x_real_score - x_fake_score)`. Should it be `d_train_model.add_loss(d_loss - grad_loss)` ?
